controller/src/main/java/io/pravega/controller/store/stream/AbstractStreamMetadataStore.java

```diff
@@ -138,15 +135,13 @@ public OperationContext createContext(String scope, String name) {
                                                    final Executor executor) {
         return withCompletion(getStream(scope, name, context).create(configuration, createTimestamp), executor)
                 .thenApply(result -> {
+                    CREATE_STREAM.reportSuccessValue(1);
```

Comment:
Is this an artifact of the long history of this PR or was there an intention to repeat these two lines twice?
```
CREATE_STREAM.reportSuccessValue(1);
DYNAMIC_LOGGER.reportGaugeValue(nameFromStream(OPEN_TRANSACTIONS, scope, name), 0);
```

Reply:
fixed